rules/no-disabled-tests.js

```diff
@@ -1,23 +1,7 @@
 'use strict';
 
 const getDocsUrl = require('./util').getDocsUrl;
-
-function getName(node) {
```

Comment:
I replaced the implementation of `util/getNodeName` with this implementation, which I think handles member expressions a little bit better than the `expect-expect` implementation.

All tests continue to pass with the refactor too. 👍